projects/core/src/global-message/http-interceptors/handlers/bad-request/bad-request.handler.ts

```diff
@@ -108,6 +109,68 @@ export class BadRequestHandler extends HttpErrorHandler {
       });
   }
 
+  protected handleDuplicated(
+    _request: HttpRequest<any>,
+    response: HttpErrorResponse
+  ): void {
+    this.getErrors(response)
+      .filter(
+        (error) =>
+          error.type === 'ModelSavingError' ||
```

Comment:
let's move them to const

Reply:
![image](https://user-images.githubusercontent.com/8040837/101636484-b0413400-3a2b-11eb-8246-0ebbd75e66bd.png)
Unfortunately we haven't implemented this good practice here, I would like to keep consistency and allow to fix that in some kind of global refactor like #8228